php/utils/file-utils.php

```diff
@@ -192,7 +192,11 @@ function ee_file_is_absolute_path( $file ) {
  * @param $replace
  */
 function ee_file_search_replace( $file, $search, $replace ) {
-	$file_content = file_get_contents( $file );
+	if(ee_file_exists($file)){
+		$file_content = file_get_contents( $file );
+	} else {
+		EE::debug("$file not found");
+	}
 	file_put_contents( $file, str_replace( $search, $replace, $file_content ) );
```

Comment:
@bnap00 Please add this line also in if condition. As if the file is exist then only we are replacing content. So It should go in `if(ee_file_exists($file)){` condition.